l2-contracts/src/L2RewardManagerStorage.sol

```diff
@@ -35,11 +35,13 @@ abstract contract L2RewardManagerStorage {
 
     /**
      * @notice Data required for bridging.
-     * @param destinationDomainId The destination domain ID.
+     * @param destinationDomainId The destination domain ID (LayerZero endpoint ID)
+     * @param endpoint The LayerZero V2 endpoint address
      */
     struct BridgeData {
         // using struct to allow future addition to this
         uint32 destinationDomainId;
```

Comment:
Double check that domains in LZ can fit uint32, if they can't this might need to change